libs/barista-components/filter-field/src/filter-field.spec.ts

```diff
@@ -232,14 +211,62 @@ describe('DtFilterField', () => {
     },
   ));
 
-  it('should focus the input field when focusing the host', () => {
-    const input = fixture.debugElement.query(By.css('.dt-filter-field-input'))
-      .nativeElement;
-    filterField.focus();
-    expect(document.activeElement).toBe(input);
+  describe('focus on input', () => {
+    beforeEach(fakeAsync(() => {
+      TestBed.configureTestingModule({
```

Comment:
you don't have to copy the same beforeEach into every describe. I guess you did that because you added another describe at the end that needs a different setup. 
You can move the last describe onto the root level - e.g. DtFilterField with FilterValueConfig on the root. 
And leave the old setup in the before each where it was.